README.md

```diff
@@ -48,3 +48,15 @@ development environment you can use.
    ```bash
    docker-compose run --rm obs-docu daps -vv -d DC-obs-all html
    ```
+
+## Update Documentation
+
+The first steps to update the documentation are:
+
+- Fork [obs-docu](https://github.com/openSUSE/obs-docu) project and clone the fork on your computer.
+- Prepare the development environment following the instructions above.
+- Add or correct a piece of documentation.
+- Commit your changes in a new branch and send a Pull Request (as in any other repository).
+- Wait until your changes are merged.
+
+The next steps should be made in _obs-landing_. Once the Pull Request is approved and merged, your changes are ready to be deployed. However, you should move to [obs-landing](https://github.com/openSUSE/obs-landing) project and deploy from there. The OBS documentation is integrated into the OBS landing page via git submodules. More details in [obs-landing README](https://github.com/openSUSE/obs-landing#updating-obs-documentation).
```

Comment:
I would write it like this:

The OBS documentation is hosted online at [openbuildservice.org](https://openbuildservice.org/help/). Instructions to deploy the OBS documentation can be found in the [obs-landing](link to instructions) repository.

And add the details of how it's integrated (git submodules) in the docu of the landing page.

Reply:
I wrote a mix between yours and mine :angel: